tests/e2e-playwright/tests/sleepers/sleepers.py

```diff
@@ -100,10 +101,11 @@ def test_sleepers(
         ),
     ):
         for _ in range(1, num_sleepers):
-            page.get_by_test_id("newNodeBtn").click()
-            page.get_by_placeholder("Filter").click()
-            page.get_by_placeholder("Filter").fill("sleeper")
-            page.get_by_placeholder("Filter").press("Enter")
+            with page.expect_response(re.compile(rf"/projects/{project_uuid}/nodes")):
```

Comment:
THOUGHT: this context manager will wait until the expected response (or times-out) and then continue with the test. Therefore, this affects somehow the timing of the test, right?  I mean, a real user does not necessary wait for the response to happen and then continue clicking around, right? 
An alternative would be to add the context outside the for loop or somewhere at even a higher level. I do not mean a better or worse alternative since we only know that that call will happen some undetermined time **after** pressing `Enter`.

Reply:
It doesn't right now, but it will after https://github.com/ITISFoundation/osparc-simcore/pull/5657